nion/swift/ExportDialog.py

```diff
@@ -273,10 +274,26 @@ def __init__(self, display_item: DisplayItem.DisplayItem) -> None:
         self.__enforce_width_height_constraints()
 
     def __calculate_display_size_in_pixels(self, display_item: DisplayItem.DisplayItem) -> Geometry.IntSize:
-        if display_item.display_data_shape and len(display_item.display_data_shape) == 2:
-            return Geometry.IntSize(height=display_item.display_data_shape[0], width=display_item.display_data_shape[1])
+        if display_item.display_data_shape and self.__display_item.used_display_type != "line_plot":
+            return Geometry.IntSize(height=display_item.display_data_shape[-2], width=display_item.display_data_shape[-1])
         return Geometry.IntSize(height=288, width=480)
 
+    @property
+    def image_info(self) -> typing.Optional[str]:
+        data_information = self.__display_item.displayed_title + "\n"
+        if self.__display_item.used_display_type != "line_plot":
+            assert self.__display_item.data_item and self.__display_item.dimensional_shape
+            calibrated_image_size = (self.__display_item.dimensional_shape[-2] * self.__display_item.data_item.dimensional_calibrations[-2].scale,
+                                     self.__display_item.dimensional_shape[-1] * self.__display_item.data_item.dimensional_calibrations[-1].scale)
+            data_information += ("(" + str(calibrated_image_size[-2]) + " " + self.__display_item.data_item.dimensional_calibrations[-2].units + "," +
+                                 str(calibrated_image_size[-1]) + " " + self.__display_item.data_item.dimensional_calibrations[-1].units + ")")
+        else:
+            assert self.__display_item.display_data_shape
+            data_information += "Line Plot Size " + str(self.__display_item.display_data_shape[0])
+            calibrated_line_plot_size = self.__display_item.display_data_shape[0] * self.__display_item.displayed_dimensional_calibrations[0].scale
+            data_information += ("(" + str(calibrated_line_plot_size) + " " + self.__display_item.displayed_dimensional_calibrations[0].units + ")")
+        return data_information
+
```

Comment:
Do we need to worry about this info changing once the window is open? If so, this will need a listener on the `property_changed_event` of `self.__display_item` and `self.__display_item.data_item` to catch any relevant changes and trigger a `property_changed_event` for `image_info`

Reply:
I suggest not worrying about this for now. It's definitely a concern but we should address it separately.